.github/workflows/static-analysis.yml

```diff
@@ -18,6 +18,8 @@ jobs:
       matrix:
         php-version:
           - "7.2"
+          - "7.4"
+          - "8.0"
```

Comment:
This is not enough to get static analysis for DBAL v3, since the ORM  does not allow it yet, and I thought that was your goal after our private chat. Can you please clarify your intentions here?

Reply:
The idea is to move one step closer to it. 
Since we were running static analysis only on 7.2, when we have added support for psr/log 2.0  (in https://github.com/doctrine/migrations/pull/1184/files) we did not notice the small imperfections about it I had to fix here